homework.md

```diff
@@ -44,70 +44,71 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language used on the client-side and server-side
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
-
+edit a commit message
+remove the file frome git whihout removing it from github
```

Comment:
Hi @Amel1983 there is typo here in the answer it says frome, should be from. 
You can also put something like GitHub solves the problem of version control and makes it easier to work collaboratively with people on projects from different parts of the work. 
Thank you.